docs/release-notes/release-notes-1.0.1.md

```diff
@@ -0,0 +1,41 @@
+# DCRDEX v1.0.1
+
+This release contains several important fixes and improvements.  All users are advised to update.
+
+Please read the [v1.0.1 release notes](https://github.com/decred/dcrdex/releases/tag/v1.0.0) for a full list of changes since v0.6.
+
+## Fixes and Improvements
+
+- Allow collapsing user-expanded reputation info on /markets
+- Removed unused changepos field in btc fundrawtransaction that was causing a JSON parse error when firod returned a negative value
+- Update minimum Firo version to 0.14.14.0
+- Add support for enabling/disabling a server
+- Fix a couple of issues with the Binance market feed subscriptions
+- Fix a lock-contention problem where a long running "tick" function was causing the client to fail to report their preimage
+- Fix high/low rate conversion on /markets
+- Fix amounts displayed for token transactions in transaction history on /wallets
+- Fix negative swap confirmations for Decred
+- Fix bond confirmation messaging on /markets that was showing two different messages at once
+- Fix filename flavor and enforce darwin cgo minimum version in bisonw-desktop builds
+- Fix double accounting for locked balance when calculating available balance in market maker bot allocations
+- Get rid of dead asset Zclassic
+- Correct market maket bot level-spacing minimum value to match precision
+- Add MATIC token on Ethereum network, for users who may have accidentally withdrawn on incorrect network
+- Fixes an persistent error for orders that were active at startup, but for which the server was no longer reachable, or the market no longer existed
+- Change native Polygon asset from MATIC to POL
+- A bunch of little UI bugs (#2968)
+- Remove too-restrictive validation of seeds (when recovering password) that didn't account for new mnemonic seeds
+
```

Comment:
Maybe it's in there but https://github.com/decred/dcrdex/commit/58a27f8c2c428d05a8883b2f44aa0c71e80421b6 prevents some dcr channels from being blocked at the end of sync